frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -56,8 +56,8 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   const regionList = await getLookupService().getAllRegions();
   const preferredLanguage = personalInformation.preferredLanguageId ? await getLookupService().getPreferredLanguage(personalInformation.preferredLanguageId) : undefined;
   const maritalStatusList = await getLookupService().getAllMaritalStatuses();
-  const dateString = personalInformation.birthDate ? new Date(personalInformation.birthDate).toLocaleDateString() : undefined;
-  const birthParsedFormat = dateString && isValid(parse(dateString, 'MM/dd/yyyy', new Date())) ? toLocaleDateString(parse(dateString, 'MM/dd/yyyy', new Date()), locale) : undefined;
+  const dateString = personalInformation.birthDate ? new Date(personalInformation.birthDate).toISOString() : undefined;
+  const birthParsedFormat = dateString && isValid(parse(dateString, "yyyy-MM-dd'T'HH:mm:ss.SSSX", new Date())) ? toLocaleDateString(parse(dateString, "yyyy-MM-dd'T'HH:mm:ss.SSSX", new Date()), locale) : undefined;
```

Comment:
I find it interesting that you're:

- converting a `string` to a `Date`
- converting that `Date` back to a `string`
- parsing that `string` using `parse()`
- converting that `string` to another `string` representation via `toLocaleDateString()`

The logic here seems unnecessarily convoluted.

Reply:
Sorry.. maybe my language was a little harsh. I'm just wondering if there's a cleaner way to do this.